charts/kong/README.md

```diff
@@ -805,6 +807,30 @@ If your SMTP server requires authentication, you must provide the `username` and
 
 By default, SMTP uses `AUTH` `PLAIN` when you provide credentials. If your provider requires `AUTH LOGIN`, set `smtp_auth_type: login`.
 
+## Prometheus Operator integration
+
+The chart can configure a ServiceMonitor resource to instruct the [Prometheus
+Operator](https://github.com/prometheus-operator/prometheus-operator) to
+collect metrics from Kong Pods. To enable this, set
```

Comment:
```suggestion
collect metrics from Kong pods. To enable this, set
```

nit - optional

Reply:
The guidance in https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects is unfortunately less than clear re the "specifically to interacting with an API object" and "generally discussing an API object" distinction, and the examples don't clarify it much--one uses lowercase "pods" but others use camel case without an interaction. IME their docs tend to camel case API objects more often than not, so I generally use that throughout. Left as-is.